README.md

```diff
@@ -28,17 +28,18 @@ Package contains Docker Compose file and configuration presets.
 Package contains several configuration presets:
  - Local - standalone masternode for local development
  - Evonet - masternode with Evonet configuration
+ - Testnet - masternode with testnet configuration
 
 There are two ways to apply a present:
  1. Rename corresponding dotenv file (i.e. `.env.local`) to `.env`
  2. Add `--env-file` option to `docker-compose` command
 
 ### Start
 
-In order to run a masternode use Docker Compose:
+In order to run a masternode use Docker Compose (optionally with evo services):
 
```

Comment:
I would add one more section above about Evo configuration and explained that there are two ways to use Evo setup:
1. Provide `-f docker-compose.yml -f docker-compose.evo.yml` every time you use `docker-compose` command
2. Create symlink `ln -s docker-compose.evo.yml docker-compose.override.yml` (or just copy?). A symlink is better from the perspective of future mn-bootstrap updates when we change evo yml

Reply:
I found a better solution with COMPOSE_FILE so we could remove all this stuff at all.